From 58a811ec1d93a68ebc83e308ef630ea81cb9c3ba Mon Sep 17 00:00:00 2001
From: Sabri Ünal <yakushabb@gmail.com>
Date: Fri, 29 May 2026 16:22:54 +0300
Subject: [PATCH] Fix build errors

Signed-off-by: Sabri Ünal <yakushabb@gmail.com>
---
 source/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/source/Makefile b/source/Makefile
index 4182c81..0e524dc 100644
--- a/source/Makefile
+++ b/source/Makefile
@@ -762,7 +762,7 @@ FREETYPE_LDFLAGS := $(shell $(PKGCONFIG) freetype2 --libs-only-L) $(shell $(PKGC
 endif
 
 ifndef BUILD_SDL2
-SDL2_INCLUDE := $(shell $(PKGCONFIG) sdl2 --cflags-only-I | sed -e 's/-I/-isystem /')
+SDL2_INCLUDE := $(shell $(PKGCONFIG) sdl2 --cflags-only-I | sed -e 's/-I/-isystem /') -isystem /app/include/SDL2
 SDL2_CFLAGS  := $(shell $(PKGCONFIG) sdl2 --cflags-only-other)
 SDL2_LDFLAGS := $(shell $(PKGCONFIG) sdl2 --libs-only-L) $(shell $(PKGCONFIG) sdl2 --libs-only-l)
 endif
@@ -773,6 +773,7 @@ LIBS += -lSDL2_mixer
 endif
 endif
 LIBS += -lSDL2_image $(SDL2_LDFLAGS) $(FREETYPE_LDFLAGS)
+LIBS += -lSDL2_image $(SDL2_LDFLAGS) $(FREETYPE_LDFLAGS)
 
 endif # pkg-config
 
--
libgit2 1.7.2

